run-tests-with-build-optawebs-parent/optawebs-project-sources-test/pom.xml

```diff
@@ -70,4 +78,42 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>openshift</id>
+            <activation>
+                <property>
+                    <name>openshift</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <executions>
+                            <execution>
+                                <id>deploy-to-openshift-and-test</id>
+                                <phase>integration-test</phase>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <executable>${project.basedir}/${project.sources.artifact}-test-script-openshift.sh</executable>
+                                    <arguments>
+                                        <argument>${sources.directory.module.project.sources.dependency.get}/${project.sources.artifact}-${download.sources.version}</argument>
```

Comment:
`${sources.directory.module.project.sources.dependency.get}` is same value now as in `${sources.directory}`. Do you want to use that property instead?

Reply:
`${project.sources.artifact}` has been only an activation property so far, it is not defined anywhere in the repository as a maven property, it might not always map to the artifactId value precisely.
Isn't it kinda surprising to rely on this in some path? I am just asking if it wouldn't be better to introduce the `project.sources.artifact` property into the top level pom? (Not sure if it does not break any enforcer rules though).

Having said all of this - it is most probably safe due to reactor order making sure this module is invoked only after the `run-tests-with-build-project-sources-dependency-get` module. And `run-tests-with-build-project-sources-dependency-get` makes sure using enforcer that `project.sources.artifact` is set properly and checked for exact value match.

I would maybe consider introducing here in this pom.xml a property that would be defined as
```
<!--
  Relying on profile activation property from run-tests-with-build-project-sources-dependency-get module.
  Enforcer of that module assures that the value is set properly.
-->
<property.name>${project.sources.artifact}</property.name>
```
and use this new property here instead (to provide some semantics behind why this actually works fine).

Reply:
> `${sources.directory.module.project.sources.dependency.get}` is same value now as in `${sources.directory}`. Do you want to use that property instead?

I prefer the explicit name here so to not mask the parameter source

Reply:
Added optaweb.project.name

Reply:
could you also add a comment to the property similarly to what I've mentioned above? So that it's clear why we can rely on this property having the value.

Reply:
Fixed